roles/cdi/defaults/main.yml

```diff
@@ -1,5 +1,5 @@
 cdi_namespace: "{{ cdi_image_namespace | default('golden-images') }}"
 cdi_storageclass: "{{ cdi_kubevirt_storageclass | default('kubevirt') }}"
 apb_action: "provision"
-repo_tag: "{{ cdi_repo_tag | default('jcoperh') }}"
-release_tag: "{{ cdi_release_tag | default('latest') }}"
+repo_tag: "{{ cdi_repo_tag | default('kubevirt') }}"
+release_tag: "{{ cdi_release_tag | default('v0.5.0-alpha.0') }}"
```

Comment:
**IF** CDI repo is really going to follow the same versions as kubevirt, i would keep only 1 param
also not sure about this default... i would prefer to not have a default and fail, than to run on a version somebody put here god knows how long ago, or would use latest and if it fails at least you know the latest version is not working

Reply:
yeah, I guess we can eliminate `cdi_repo_tag` by hard coding it into template, but I guess this is also fine ? I would like to keep it cuz sometime it maybe more easy for debugging, but for this case since there's only one yaml file here, so I guess I don't have strong opinion on keep one parameter or two.

> also not sure about this default... i would prefer to not have a default and fail, than to run on a version somebody put here god knows how long ago, or would use latest and if it fails at least you know the latest version is not working

Yeah, this is what I suggested using latest and I filed an issue here: https://github.com/kubevirt/containerized-data-importer/issues/177, but seems they didn't use the `latest` tag on purpose.